eq/client/glx/pipe.cpp

```diff
@@ -261,6 +260,10 @@ bool Pipe::_configInitGLXEW()
     {
         LBVERB << "Pipe GLXEW initialization successful" << std::endl;
         success = configInitGL();
+
+        const char* glVersion = (const char*)glGetString( GL_VERSION );
```

Comment:
success is user-code specific. When here, success is true already, indicating successful init of GL and GLEW. LBVERB here also says so :)